qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ConfigureMetadataPage.java

```diff
@@ -15,6 +15,7 @@
 public class ConfigureMetadataPage extends Page {
 
 	protected static final String PAGE_URL = "adminui/metadata/configureMetadata.page";
+	private static final By MANAGE_PRIVILEGES = By.cssSelector("#org-openmrs-module-adminui-Privileges-org-openmrs-module-adminui-Privileges-extension"); 
```

Comment:
is `MANAGE_PRIVILEGES` a button or link?

Reply:
@mherman22 did you take a look at the comment [above](https://github.com/openmrs/openmrs-contrib-qaframework/pull/270/files#r768754419)?